e2e/uusi_hanke.spec.ts

```diff
@@ -1,110 +1,150 @@
-    await page.getByTestId('draw-control-Polygon').click();
-    await page.locator('canvas').first().click({ position: { x: 454, y: 221, } });
-    await page.locator('canvas').first().click({ position: { x: 543, y: 221, }, });
-    await page.locator('canvas').first().click({ position: { x: 454, y: 289, } });
-    await page.locator('canvas').first().click({ position: { x: 454, y: 221, }, });
-    await page.getByLabel('Valitse päivämäärä').first().click();
-    await page.getByRole('button', { name: `${testiData.currentMonth} ${testiData.todayDate}`, exact: true }).click();
-    await page.getByLabel('Haittojen loppupäivä*').click();
-    await page.getByLabel('Haittojen loppupäivä*').fill(testiData.tomorrowType);
-    await page.getByRole('button', { name: 'Meluhaitta *' }).click();
-    await page.getByRole('option', { name: 'Satunnainen meluhaitta' }).click();
-    await page.getByRole('button', { name: 'Pölyhaitta *' }).click();
-    await page.getByRole('option', { name: 'Satunnainen pölyhaitta' }).click();
-    await page.getByRole('button', { name: 'Tärinähaitta *' }).click();
-    await page.getByRole('option', { name: 'Satunnainen tärinähaitta' }).click();
-    await page.getByRole('button', { name: 'Autoliikenteen kaistahaitta *' }).click();
-    await page.getByRole('option', { name: 'Yksi autokaista vähenee -' }).click();
-    await page.getByLabel('', { exact: true }).click();
-    await page.getByRole('option', { name: 'Alle 10 m' }).click();
-    await page.getByRole('button', { name: 'Seuraava' }).click();
-    await page.getByText('Hakemus tallennettu').waitFor({ state: 'hidden', timeout: 10000 });
+  test.setTimeout(240000);
+  await page.getByLabel('Luo uusi hanke.', { exact: true }).click();
+  await page.getByTestId('nimi').click();
+  const ajonNimi = hankeName(`uusi-hanke`);
+  await page.getByTestId('nimi').fill(ajonNimi);
+  await page.getByTestId('perustaja.sahkoposti').click();
+  await page.getByTestId('perustaja.sahkoposti').fill(perustaja.email);
+  await page.getByTestId('perustaja.puhelinnumero').click();
+  await page.getByTestId('perustaja.puhelinnumero').fill(perustaja.phonenumber);
+  await page.getByRole('button', { name: 'Luo hanke' }).click();
+  await page.getByTestId('kuvaus').click();
+  await page.getByTestId('kuvaus').fill(`${ajonNimi} Tämä on testiautomaatiota varten luotu`);
+  await page.getByTestId('tyomaaKatuosoite').click();
+  await page.getByTestId('tyomaaKatuosoite').fill(testiOsoite.address);
+  await page.getByText('Ohjelmointi').click();
+  await page.getByLabel('', { exact: true }).click();
+  await page.getByLabel('Työn tyyppi: Sulje ja avaa').click();
+  await page.getByRole('option', { name: 'Vesi', exact: true }).getByLabel('check').click();
+  await page.getByLabel('Työn tyyppi: Sulje ja avaa').click();
+  await page.getByRole('button', { name: 'Seuraava' }).click();
+  await page.getByText('Hanke tallennettu').waitFor({ state: 'hidden', timeout: 10000 });
```

Comment:
I think the logic from `johtoselvitys_ja_liite_hankkeelle.spec.ts` applies here as well. The wait is clearly unnecessary since it hasn't done anything so far.

If anything is done to these, I think they should be changed follow the logic from `johtoselvitys_ja_liite_hankkeelle`:
```typescript
  await expect(page.getByText('Hanke tallennettu')).toBeVisible();
  await page.getByRole('alert').getByLabel('Close toast', { exact: true }).click({ timeout: 2000 });
```